shared/guide/system/auditing/auditd_configure_rules/audit_privileged_commands/audit_rules_privileged_commands_newgrp.rule

```diff
@@ -0,0 +1,49 @@
+documentation_complete: true
+
+prodtype: rhel7
```

Comment:
Can this field be easily expanded to accommodate other products?

And in case the Rule is also applicable to other products, if we would like to have different description, would it be like `description@otherproduct:`?

Reply:
The idea is to use jinja for that in the future, right now I will just port exactly what we have to yaml.

THis is how it could look in the future:
```
title: blabla
{% if prodtype=rhel7 %}
description: |
     something somethign rhel specific
{% else %}
description: |
    something somethign generic
{% endif %}
```